src/Variables/MultipleChoiceVariable.php

```diff
@@ -82,8 +82,10 @@ public function getValue(RecordInterface $record): ValueInterface|ValueSetInterf
 
     public function getDisplayValue(RecordInterface $record, ?string $locale = null): StringValueInterface
     {
-        /** @var ValueSetInterface $valueSet */
         $valueSet = $this->getValue($record);
+        if (!$valueSet instanceof ValueSetInterface) {
+            return new MissingStringValue((string) $valueSet->getRawValue(), $valueSet->isSystemMissing());
+        }
```

Comment:
@radwouters these are the kinds of errors you hide when using type assertions to bypass the static analyser.